packages/server/modules/gatekeeperCore/domain/billing.ts

```diff
@@ -1,29 +1,27 @@
 import { z } from 'zod'
 
-// team
-export const trialWorkspacePlans = z.literal('starter')
+const trialWorkspacePlans = z.literal('starter')
```

Comment:
1. i removed export where unused, this makes it easier to understand which are important globals and which are just implementation details for other globals in the file. easier to reason about
2. added *Schema suffix for some of the zod schemas, cause otherwise TS gets pissy about them being unused other than in `typeof X` TS expressions (which ultimately get stripped). the *Schema suffix is an override for the TS rule